environmental-values.csl

```diff
@@ -128,10 +131,6 @@
     <names variable="author">
       <name delimiter-precedes-last="always" initialize-with="." name-as-sort-order="all"/>
       <label form="short" prefix=". "/>
-      <substitute>
```

Comment:
Not sure why you're deleting the substitute? The output here seems wrong (see the two items now starting with the date in the automatically generated examples)

Reply:
Sorry, I am not sure what <substitute> does. I can look. I was using the visual editor and the problem was the double full stops, I think in some reference types.

Reply:
They'd be unrelated. The substitute element ensures that the editor or title are used in place of the author if not author is present e.g. for an edited volume (editor) or a webpage (title). Almost certain this should just go back in.

Reply:
The label gets inhertited from above, no need to include it as a suffix, that's why I deleted.